stdlib/_socket.pyi

```diff
@@ -692,3 +692,28 @@ if sys.platform != "win32" or sys.version_info >= (3, 8):
     def if_nameindex() -> list[tuple[int, str]]: ...
     def if_nametoindex(__name: str) -> int: ...
     def if_indextoname(__index: int) -> str: ...
+
+if sys.version_info >= (3, 12):
```

Comment:
Do these need to be re-exported from socket.pyi as well?

Reply:
Yes, because in runtime all is re-exported, good catch!